jenkins/stage-maven-release.jenkinsfile

```diff
@@ -0,0 +1,85 @@
+lib = library(identifier: 'jenkins@main', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build.git',
+]))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2'
+            args '-e JAVA_HOME=/opt/java/openjdk-11'
+            alwaysPull true
+        }
+    }
+    environment {
+        VERSION = "${params.VERSION}"
+        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/client/opensearch-java/${VERSION}"
+    }
+    stages {
+        stage('parameters') {
+            steps {
+                script {
+                    properties([
+                        parameters([
+                            string(
+                                name: 'REF',
+                                trim: true
+                            ),
+                            string(
+                                name: 'VERSION',
+                                trim: true
+                            )
+                        ])
+                    ])
+                    if (params.REF.isEmpty() || params.VERSION.isEmpty()) {
+                        currentBuild.result = 'ABORTED'
+                        error('Missing REF and/or VERSION.')
+                    }
+                }
+            }
```

Comment:
do you want to add triggers now or I can create a follow up PR. I have the code ready. Needs few PRs to add creds to jenkins infrastructure.

Reply:
Let's do a followup.